portality/bll/services/todo.py

```diff
@@ -19,7 +19,7 @@ def group_stats(self, group_id):
         for editor in editors:
             acc = models.Account.pull(editor)
             stats["editors"][editor] = {
-                    "email" : acc.email
+                    "email" : None if acc is None else acc.email
```

Comment:
Are there any downstream consequences of this that we need to account for?

Reply:
There are no immediate downstream consequences. Still, other parts of the system will try to use the email of the non-existent account, but finding those would be part of a general fix, not that workaround.